engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/hydration/NadelHydrationFieldsBuilder.kt

```diff
@@ -63,18 +61,18 @@ internal object NadelHydrationFieldsBuilder {
         val fieldChildren = deepClone(fields = hydratedField.children)
             .mapNotNull { childField ->
                 val objectTypesAreNotReturnedByActorField =
-                    actorFieldOverallObjectTypeNames.intersect(childField.objectTypeNames).isEmpty()
+                    actorFieldOverallObjectTypeNames.none { it in childField.objectTypeNames }
```

Comment:
Refactored this to `none` instead because that avoids creating a new collection just to check `.isEmpty` on it.